codalab/templates/su/login_form.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'base.html' %}
+
+{% load switchuser %}
+{% load url from future %}
+
+{% block head_title %}Switch-User{% endblock %}
+
+{% block content %}
+    {% if not su_state %}
+        <div class="alert alert-danger">
+            Error: the <tt>su_state</tt> context processor was not found. Is
+            <tt>"django_switchuser.context_processors.su_state"</tt> in your list
+            of <tt>TEMPLATE_CONTEXT_PROCESSORS</tt>?
+        </div>
+    {% else %}
+        {% if su_state.is_active %}
+            <div class="alert alert-success">
+                Currently logged in as: <em>{% su_user_short_label su_state.active_user %}</em>
+            </div>
+        {% endif %}
+        <div align="center" class="well">
+            <form method="POST" action="{% url "su-login" %}?next={{ next }}">
+                {% csrf_token %}
+                <label for="user_id">Login as:</label>
+                <br>
```

Comment:
try not to use `<br>` and `<center>` and such... use `<p>` maybe? also looks like indentation here is off a bit?
![image](https://user-images.githubusercontent.com/2185159/37795633-49655eea-2dd2-11e8-8bc0-acd3a9b92636.png)

Reply:
(not really too big a deal for this page, but in the future)